lib/project_types/script/forms/script_form.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+module Script
+  module Forms
+    class ScriptForm < ShopifyCli::Form
+      protected
+
+      def organization
+        @organization ||= ask_organization(ctx)
+      end
+
+      def organizations
+        return @organizations if defined?(@organizations)
+        UI::StrictSpinner.spin('Fetching organizations') do |spinner|
+          @organizations = ShopifyCli::PartnersAPI::Organizations.fetch_with_app(ctx)
+          spinner.update_title('Fetched organizations')
+        end
+        @organizations
+      end
+
+      def ask_app_api_key(ctx, apps, message: 'Which app do you want this script to belong to?')
+        if apps.count == 0
+          raise Errors::NoExistingAppsError
+        elsif apps.count == 1
+          ctx.puts("Using app {{green:#{apps.first['title']} (#{apps.first['apiKey']})}}.")
```

Comment:
@tanema FYI we also log the auto-selected option.  Might be something different here in Script land.